docs/user-guide/SettingsFile.md

```diff
@@ -37,6 +37,72 @@ Path to your key file in a txt file.
 
 If provided and valid, RESTler will attempt to use it during the SSL handshake.
 
+### authentication: dict (default empty)
+Settings for specifying authentication. See Authentication.md for details
+
+__Token__: Can optionally provide one of {```location```, ```token_refresh_cmd```, ```module```}
+
+__location str (Default None)__: File path to a text file containing a token
+
+```json
+"authentication": {
+    "token": {
+      "location":, "/authentication_token.txt"
+      "token_refresh_interval":  300
+    }
+}
+```
+
+__token_refresh_cmd str (Default None)__: The command to execute in order to refresh the authentication token
+
+```json
+"authentication": {
+    "token": {
+      "token_refresh_cmd": "python unit_test_server_auth.py"
+      "token_refresh_interval": 300
+    }
+}
+```
+
+__module json (Default None)__: Dictionary containing settings for RESTler to invoke user-specified module to refresh the authentication token
```

Comment:
>json [](http://example.com/codeflow?start=9&length=4)

for consistency with the rest of the file, could you please declare all of these as 'dict' instead of 'json'?